src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/utils/screenshot/ScreenshotUtils.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.commonUiTestLibrary.utils.screenshot;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import com.intellij.remoterobot.RemoteRobot;
+
+/**
+ * Static utilities for taking screenshots
+ *
+ * @author zcervink@redhat.com
+ */
+public class ScreenshotUtils {
+    /**
+     * Take screenshot of the entire screen and save it on disk
+     *
+     * @param remoteRobot reference to the RemoteRobot instance
+     * @return            the screenshot as a File object
+     */
+    public static File takeScreenshot(RemoteRobot remoteRobot) {
+        final String screenshotLocation = "./build/screenshots/";
```

Comment:
Is this working also on Windows?

Reply:
Well I see that slashes there... According to the ScreenshotUtilsTest it works on all platforms, but File.separator would be better approach here. Fixed.